src/utils/helpers/review/getAssignedQuestions.ts

```diff
@@ -3,35 +3,34 @@ import {
   ReviewQuestionAssignment,
   TemplateElement,
 } from '../../generated/graphql'
-import { ReviewQuestion } from '../../types'
 interface GetAssignedQuestionsProps {
   reviewQuestions: ReviewQuestionAssignment[]
 }
 
+/**
+ * @function getAssignedQuestions
+ * Returns array of type ReviewQuestion with assigned questions
+ * to Reviewer from fetched in GraphQL type ReviewQuestionsAssigment
+ * @param reviewQuestions GraphQL array of assigned questions
+ */
 const getAssignedQuestions = ({ reviewQuestions }: GetAssignedQuestionsProps) => {
-  const assignedQuestions = reviewQuestions.reduce(
-    (validQuestionAssignments: ReviewQuestion[], questionAssignment) => {
+  const assignedQuestions = reviewQuestions
+    .filter((questionAssignment) => {
+      // Remove review_response without a response link
+      const { templateElement } = questionAssignment
+      const { applicationResponses } = templateElement as TemplateElement
+      return applicationResponses.nodes.length > 0
```

Comment:
Just wondering why there'd be review responses without a link to an application response?

Reply:
It's more like a safeguard for now. In case there is one, it wouldn't be considered when calculating the progress for example. We would also need some improvement here when we are dealing with Consolidation `review_responses` which wouldn't have the `application_response` link.